docs/sequenceDiagrams/ListSequenceDiagramCode.txt

```diff
@@ -0,0 +1,31 @@
+title List
```

Comment:
Nicely done. one small edit will be making the activation bar for the ListCommand
avtivate ListCommand